examples/bucket_schemas.py

```diff
@@ -0,0 +1,95 @@
+"""
+This example is related to `InfluxDB Cloud <https://docs.influxdata.com/influxdb/cloud/>`_ and not available
+on a local InfluxDB OSS instance.
```

Comment:
Do we know if this is in or not in Enterprise as well?

Reply:
According to the documentation, this is not available in InfluxDB Enterprise.

- https://docs.influxdata.com/enterprise_influxdb/v1.10/tools/api/